app/javascript/components/CardColumn/Card/CommentsDropdown/index.js

```diff
@@ -65,6 +65,10 @@ const CommentsDropdown = ({id, comments}) => {
     } else if (evt.key === 'Enter' && evt.ctrlKey) {
       handleSubmit(evt);
     }
+
+    if (evt.key === 'Escape') {
```

Comment:
I think it's a good place for implementing https://github.com/cybergizer-hq/retrospective/issues/169